tap_linkedin_ads/__init__.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 
+# type: ignore
```

Comment:
what is stubs?

Reply:
stubs?

Reply:
ahh, the commit name :) . Stubs is something that you can do to wrap some code such that is has typing information. Basically what you do if you are, say, mixing normal javascript and typescript. You will then create a wrapper for the non-typed javascript that is typed, and then call the wrapper from typescript and everyone is happy - although a bit confused.